guides/common/modules/proc_using-tftp-services-through-nat.adoc

```diff
@@ -18,12 +18,9 @@ endif::[]
 ----
 include::snip_make-firewall-settings-persistent.adoc[]
 
-ifdef::satellite[]
-.Using NAT on {RHEL} 6:
-endif::[]
 ifndef::satellite[]
 .Using NAT on linux with `iptables`:
```

Comment:
Should I remove this whole procedure with `iptables`?

Reply:
I'm questioning the whole procedure. Already did so when looking at the new provisioning infrastructure services guide, but didn't want to blow up the scope.

I don't quite know what the point of the whole chapter is. I'd be tempted to just drop it altogether so I've opened https://github.com/theforeman/foreman-documentation/pull/3852.